coalib/settings/ConfigurationGathering.py

```diff
@@ -138,8 +138,13 @@ def load_config_file(filename, log_printer=None, silent=False):
                                           file=filename,
                                           found='does not exist'))
                 sys.exit(2)
+    except PermissionError:
+        if not silent:
+            logging.error('coala doesn\'t have permission to read
```

Comment:
mistake -->`doesn\'t`
and put `'` in the end of string in line 143 and in the start of string in line 144. It will solve the `CI` error.

Always try to read first,before asking for `review` sometimes googling of error can help in getting the problem.
Here is the [answer](https://stackoverflow.com/questions/3561691/python-syntaxerror-eol-while-scanning-string-literal)

Reply:
I tried reading but couldn't understand. I apologise for it. @RaiVaibhav